packages/teams-js/updateBlazorApp.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
```

Comment:
No need to copy new build of `teams-js` file to `blazorApp` directory. Is there any other way to work around this?

Reply:
see mega-comment